### DISCUSSION TITLE
PHP 7.4 support

### DISCUSSION THREAD

**Discussion by medilies** [PHPOffice/PhpSpreadsheet#3636] [General]
Shouldn't the package start requiring PHP 8.0 ?

**Comment by MarkBaker** [comment]
The package works with PHP 8.0, 8.1, 8.2 and with the daily 8.3 build.

If you mean we should drop support for 7.4, you're right. See [Issue #3634](https://github.com/PHPOffice/PhpSpreadsheet/issues/3634)
Support for 7.4 will be dropped with the next release.

**Reply by medilies** [reply]
That's exactly what I meant, thanks for the prompt response.
